src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -1399,25 +1399,43 @@ bool NetworkOPsImp::checkLastClosedLedger (
 }
 
 void NetworkOPsImp::switchLastClosedLedger (
-    Ledger::pointer newLedger, bool duringConsensus)
+    Ledger::pointer newLCL, bool duringConsensus)
```

Comment:
Every place I ever worked it was policy not to make cosmetic changes during bugfixes - to keep "renames and whitespace" away from "critical algorithm changes" to make for better auditing of changes.

Perhaps you're smarter than I am, but I just want to see the functional changes and not have them mixed with cosmetic changes.  All this bulk interferes with my ability to correctly process the diffs.

In particular, changing `openLedger` to `newOL` is obscure - and `newLCL` is also obscure.  What's wrong with `openLedger` and `lastClosedLedger`?

I know you don't like comments like this.  If you don't want style comments in a bugfix review, don't make style changes in a bugfix!